src/domain.rs

```diff
@@ -48,6 +53,7 @@ pub struct Domain<F> {
     nbulk_reclaims: AtomicUsize,
     count: AtomicIsize,
     shutdown: bool,
+    shard_hash_state: AtomicUsize,
```

Comment:
I don't think this change is relevant to this PR?

Reply:
Oops, good catch